### ISSUE TITLE
Define location where the workouts should be stored

### ISSUE THREAD

**Issue by Armagedoom** [tijmenvangulik/Ergometerspace#9]
### Motivation

This would allow the users to move the default storage folder of the workouts into their cloud-storage folder, so there is no way they can loose them.

At the moment this functionality can be replaced by manually exporting the workout after every workout into the desired folder, maybe its enough as it is, I leave the decision to you :)

Thanks for this wonderful tool!

**Comment by tijmenvangulik** [comment]
Yes that would be a great feature. When running ergometer space in the browser, local storage is not the safest place.

**Comment by tijmenvangulik** [comment]
What needs to be done is to move all items from the local storage to another storage. When you import it you first set all the items and then you need to do a refresh. This fixes also the multi user problem with import /export to the local storage all user data is changed. You can write a plugin for this if you want. (there is little knowledge needed on ergometer-space it selves, it is just local storare /external storage and make some small html interface)

**Comment by Armagedoom** [comment]
Im afraid I dont get what you mean. 
Also, Im using the installed version so this might not apply.
Not sure if you are referring to a way to do this manually. My first idea was to move the installation folder into the cloud storage folder, but when checking it I couldn't find any recently edited files that would contain the workouts. Thats when I decided to put this request.
Also, could I ask where are the workout files stored when the app is installed in Windows?

**Comment by tijmenvangulik** [comment]
I think this was I bit of “web developer language”, I assumed that you are a developer. But never mind, the feature is not there, and should be written by a developer. It can be on the list. You can read this article on where chrome is actually storing the local storage data:

https://www.howtogeek.com/255653/how-to-find-your-chrome-profile-folder-on-windows-mac-and-linux/

So every windows/mac user has its own local storage, and thus its own ergometer-space data. So switching the windows/mac user will switch the data.

But I understand your feature that you want to switch user and store the data on a better location. So this is on the wish list.

Tijmen
